packages/orchestrator-ui-components/src/components/WFOForms/UserInputFormStyling.ts

```diff
@@ -0,0 +1,89 @@
+// Todo: theme with decent colors from theme (style was copied from v1)
+const PRIMARY_COLOR = '#0077cc';
+const LIGHT_PRIMARY_COLOR = '#520980';
+
+import { css } from '@emotion/react';
+
+export const userInputFormStyling = css`
```

Comment:
A few generic comments about styling:
- In the rest of the library we are using the css object way of specifying the styles, that also solves getting the theme into the styles.
- When a component has multiple files (for example styles and later storybook) you might want to move each component in its own folder -- at least the ones containing multiple files